go/rtl/auth.go

```diff
@@ -15,6 +15,12 @@ import (
 	extra "github.com/json-iterator/go/extra"
 )
 
+func init() {
```

Comment:
Is there an example or documentation of how this is to be called? 

Seems a little magic but I haven't pulled the code to look at the details of what is going on.

Reply:
It's called once on the package being imported. 
https://medium.com/golangspec/init-functions-in-go-eac191b3860a
https://www.digitalocean.com/community/tutorials/understanding-init-in-go

Reply:
TIL about init in go!!!!!

Thanks for the info.